src/open_inwoner/react/components/SideNav/SideNav.spec.tsx

```diff
@@ -6,7 +6,7 @@ import SideNav from './SideNav'
 import '@testing-library/jest-dom'
 
 // Mock MaterialIcon to simplify rendering and testing
-vi.mock('../MaterialIcons/MaterialIcons', () => ({
+vi.mock('@react/components//MaterialIcon/MaterialIcon', () => ({
```

Comment:
Not sure if this is the correct source - it might be:

```suggestion
vi.mock('@react/components/MaterialIcon', () => ({
```

Reply:
Find-&-replace gone wrong! Funny how it still passed the tests?